pages/shows/index.vue

```diff
@@ -33,36 +33,21 @@ export default {
     }
   },
   computed: {
-    arcsiShows: {
-      get () {
-        return this.$store.state.arcsiShows
-      },
-      set (newValue) {
-        this.arcsiShows = [newValue]
-      }
-    },
     arcsiShowsList () {
       if (this.arcsiShows) {
-        const arcsiShowsList = [...this.arcsiShows]
-        return arcsiShowsList.filter((show) => {
-          return !(show.archive_lahmastore_base_url === 'off-air' || !show.active)
-        }).sort((a, b) => a.name.localeCompare(b.name))
+        return this.arcsiShows.filter(show => (
```

Comment:
Thanks for simplifying this and putting the State in one variable, nice implementation!